packages/field/src/react/__stories__/multi-select-field.story.tsx

```diff
@@ -0,0 +1,293 @@
+import { Meta, Story } from '@storybook/react/types-6-0'
+import Button from '@pluralsight/ps-design-system-button'
+import { layout } from '@pluralsight/ps-design-system-core'
+import { CaretDownIcon, CloseIcon } from '@pluralsight/ps-design-system-icon'
+import Tag from '@pluralsight/ps-design-system-tag'
+import { HTMLPropsFor } from '@pluralsight/ps-design-system-util'
+import { useCombobox, useMultipleSelection } from 'downshift'
+import { css } from 'glamor'
+import React, {
+  ChangeEventHandler,
+  ComponentProps,
+  MouseEvent,
+  MouseEventHandler,
+  forwardRef,
+  useCallback,
+  useMemo,
+  useState
+} from 'react'
+
+import { Option, periodicElements } from '../__fixtures__/options'
+import Field from '..'
+
+const GUTTER_SIZE = 4
+
+interface MultiSelectFieldProps extends ComponentProps<typeof Field> {
+  options: Option[]
+  placeholder?: string
+}
+const MultiSelectField: React.FC<MultiSelectFieldProps> = props => {
+  const { options, disabled, placeholder, ...rest } = props
+
+  const [filterTerm, setFilterTerm] = useState<string>('')
+  const handleFilterTermChange: ChangeEventHandler<HTMLInputElement> = useCallback(
```

Comment:
good catch. i need to do some investigation into what should be optimized. at least for these stories they are just to proved the approach and if/when we used a similar approach for an official component we can invest the research time them. i think it's best to remove the callbacks since i have no indication they help. i'll do that now